base/abstractarraymath.jl

```diff
@@ -84,8 +84,39 @@ function _dropdims(A::AbstractArray, dims::Dims)
     end
     reshape(A, d::typeof(_sub(axes(A), dims)))
 end
+
 _dropdims(A::AbstractArray, dim::Integer) = _dropdims(A, (Int(dim),))
 
+"""
+    dropdims(f, args...; dims, kwargs...)
+
+Compute reduction `f` over dimensions `dims` and drop those dimensions from the result.
+
```

Comment:
```suggestion
The reducer `f` must accept a `dims` keyword argument, and must naturally reduce the given dimension(s) to have size 1 (i.e. must make them singleton dimensions).
Examples of valid `f` include `sum` and `Statistics.mean`,
but do not include `sort` which while it accepts a `dims` keyword argument does not perform a reduction.
```

Reply:
Thanks - yes, I think good to be more explicit about `f`.

I think examples are best left to the actual code examples (e.g. I don't think the `sort` not being vliad is at all abiguous here)